### DISCUSSION TITLE
maxwellinked.eth hacked 6/10/2023

### DISCUSSION THREAD

**Discussion by timmaxwell** [blockscout/blockscout#9454] [Support]
Is this wallet I thought was hacked, truly hacked? Can I still claim as the rightful owner? Please email? And, thank you

**Comment by ekawibisono** [comment]
Hello, This is blockscout github, please report on security public. thanks

**Comment by Muhsanshu** [comment]
Someone hacks my account and transferred my Sidra and tokens. Old how can I get my Sidra tokens back

**Reply by Zakriya04** [reply]
Same my account got also hacked. He stole my SDA
Is this site got compromised

**Comment by Zakriya04** [comment]
Someone hacked my sidra account and stole token I didn't share with anyone or didn't click on any type of link 
I think this site got compromised

**Reply by GimluCom** [reply]
This is the discussions page for the open-source Blockscout explorer. For your issue you should contact the chain or platform that you used
